tests/conftest.py

```diff
@@ -124,14 +121,28 @@ def ecs_client_execute_run_details_success(
     mock_boto3_client,
 ):
     mock_boto3_client.run_task.return_value = boto3_ecs_client_run_task_response_success
-
     with patch(
-        "webapp.utils.aws.ECSClient.task_exists"
+        "webapp.utils.aws.ECSClient.task_definition_exists"
     ) as mock_webapp_ecs_client_task_exists:
         mock_webapp_ecs_client_task_exists.return_value = True
         yield
 
 
+@pytest.fixture
+def ecs_client_get_active_tasks_success(
+    boto3_ecs_client_describe_task_response_success, mock_boto3_client
+):
+    mock_boto3_client.describe_tasks.return_value = (
+        boto3_ecs_client_describe_task_response_success
+    )
+
+
+@pytest.fixture
+def boto3_ecs_client_describe_task_response_success():
```

Comment:
Liking this fixture, and how it's used in the test, as it helps explain the parsing in `ECSClient.get_active_tasks()`.